common/static/common/css/base.css

```diff
@@ -111,9 +107,9 @@ input[type="color"],
     font-size: 13px;
     font-weight: normal;
     line-height: 1;
-    -webkit-border-radius: 15px;
-    -moz-border-radius: 15px;
-    border-radius: 15px;
+    -webkit-border-radius: 3px;
+    -moz-border-radius: 3px;
```

Comment:
Btw you *really* don’t need all these prefixes any more. Not only is the unprefixed version supported everywhere, but `-moz-` is for Firefox, and this is a Chrome extension.